<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminClient</class>
 <widget class="QDialog" name="AdminClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Client</string>
  </property>
  <widget class="QPushButton" name="pushButton_QuitApplication">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>561</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit Application</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_RaffleEntries">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>341</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DeleteEntry">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveEntries">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Entries</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_winnerDisplay">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_chooseWinner">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Randomize and Choose One Winner</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>280</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose how many winners there are:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAmountOfWinners">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExportWinners">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Amount of Winners to txt File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of entries:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_numOfEntries">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>430</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
